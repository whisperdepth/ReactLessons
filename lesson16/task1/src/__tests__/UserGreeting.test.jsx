import React from "react";
import { shallow } from "enzyme";
import UserGreeting from "../UserGreeting.jsx";

describe("UserGreeting", () => {
  it("should display user greeting", () => {
    const wrappedComponent = shallow(<UserGreeting />);

    expect(wrappedComponent).toMatchSnapshot();
  });
});