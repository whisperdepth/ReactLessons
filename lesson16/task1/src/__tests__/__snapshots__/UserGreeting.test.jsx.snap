// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserGreeting should display user greeting 1`] = `
<h1>
  Hello, user ðŸ˜Ž
</h1>
`;
